lightning/src/ln/msgs.rs

```diff
@@ -1590,6 +1592,8 @@ pub trait OnionMessageHandler: EventsProvider {
 	/// Handle an incoming `onion_message` message from the given peer.
 	fn handle_onion_message(&self, peer_node_id: &PublicKey, msg: &OnionMessage);
 
+	fn handle_onion_message_response<T: OnionMessageContents>(&self, response: T, reply_path: BlindedPath, log_suffix: fmt::Arguments);
+
```

Comment:
Adding this to the trait is no longer necessary.

Reply:
Moved the fn back to it's original position in [2806122](https://github.com/lightningdevkit/rust-lightning/pull/2907/commits/2806122f65d57d3a6a91591aff6e9ccb7590af13)
Thanks!